app/code/community/Ebizmarts/MailChimp/etc/system.xml

```diff
@@ -133,7 +135,8 @@
                             <show_in_default>1</show_in_default>
                             <show_in_website>1</show_in_website>
                             <show_in_store>1</show_in_store>
-                            <comment><![CDATA[In case no Interest groups are selected it will work as a normal subscription to the configured audience]]></comment>
+                            <comment>
```

Comment:
This blank lines after the start tag cause failure.

Reply:
Solved.